include/eosio/vm/execution_context.hpp

```diff
@@ -24,6 +24,13 @@
 #include <system_error>
 #include <utility>
 
+#ifdef __APPLE__
+#ifndef _XOPEN_SOURCE
+#define _XOPEN_SOURCE 700
```

Comment:
Could you add a comment indicating which posix functions you are pulling in with this.

Reply:
`ucontext.h` can't be included at all without it.